clang/File.d

```diff
@@ -13,8 +13,13 @@ struct File
 {
     mixin CX;
 
-    @property string name ()
+    string name() const
     {
-        return toD(clang_getFileName(cx));
+        return toD(clang_getFileName(cast(void*) cx));
```

Comment:
Currently `File::toString()` and because I made it const and yes currently it const can be removed from both methods. But it is only a matter of time when someone needs to call that method on const File. For now I can remove it.